README.md

```diff
@@ -44,15 +37,33 @@ module "rad-security-connect" {
 
 Once applied, Rad Security will start synchronizing your resources with our platform.
 
-## Providing a Service Principal
+## Using an Existing Service Principal
+
+It is possible to use an existing Service Principal that has the necessary permissions. To do this, you can register the Azure Subscription and Tenant by creating a `azure_register` resource with the rad-security provider. There is no need to use this module. The Service Principal ID and Secret will be securely stored in the RAD Security platform. A new resource has to be created for each Azure Subscription and Tenant you wish to connect to RAD Security.
```

Comment:
Can I clarify what is meant with the `There is no need to use this module` comment?

Reply:
The module creates a bunch of resources. A role, a service principal associated to our Enterprise Application, and a role assignment. If a customer already has a Service Principal they want us to use, there is no need for any of those resources. They can register it directly to us through our provider.

Reply:
Which is the "this" in "this module"?  `resource "rad-security_azure_register"`, or one of the other modules?  It seems like we're referring to another module, and register is facilitating the workflow you mentioned above.  If it is `azure_register`, then we do need it, right?  I'm just having trouble understanding the comment.

Reply:
The wording is difficult and confusing. I pushed a new version.